_posts/2018-06-25-java-releases.md

```diff
@@ -0,0 +1,85 @@
+
+![Java release train]({{ "/assets/posts/java-release-train/roadmap-java.png" | absolute_url }})
+
+XXXXXX BILD XXXXXXXX roadmap-java.png
+
+As you can see in the diagram the free support of a Java version was quite long and the timespan in that 2 
+versions were supported in parallel was long enough to plan and handle a migration of your software to the 
+newest Java version. Even if this period was to short based on some constraints it was possible to extend it 
+by buying commercial support for a Java version.
+
+As already said all this points will change in future based on that the new release train for Java. The 
+following image shows the release train as it was announced by Oracle.
+
+XXXXXX BILD XXXXXXXX
+
+Starting with Java 9 mostly all Java version will only have a lifetime of 6 month. After this periode no more 
+updates will be provided for the version. Even if you have a commercial contract with Oracle you won’t get 
+any additional updates for Java 9 or 10 after this 6 months. Next to this all XX years a LTS ( long term 
+support) version of Java will be released. The first version with LTS support will be Java 11 that is 
+scheduled for this autumn. This LTS version will provide a commercial support with updates for XX years. The 
+most important fact is that updates for LTS version can only be accessed and used by people that buy 
+commercial support at Oracle. Based on my daily work I would say that most companies that create software 
+based on Java do not have commercial support today since the release train of Java was quite „friendly“ and 
+an adoption of the newest version was easily possible.
+
+## Why is Oracle doing this?
+TODO: ASChneller neue Features, Geld verdienen
+
+## What does the new release train mean to my company?
+If we concentrate on the Oracle JDK the answer to that question is quite easy and you can choose between 3 
+options:
+- Update the Java version all 6 month. By doing so you will always depend on the supported Java version and 
+will automatically get all important feature and security updates.
+- Buy commercial support from Oracle and migrate only from one LTS version to the next LTS version. This 
+would mean that you migrate from Java 8 to Java 11 maybe within the next year and than from Java 11 to Java 
+XX in XXXX.
+- Stay on a Java version that is not supported anymore. Once the free support of a Java version ends nobody 
+forbids you to use the unsupported version in future. By doing so you do not need to pay commercial support 
+or get into a maybe stressfull migration all 6 month. While you can now easily decide on your own when you 
+want to migrate to a newer Java version you will miss security updates and bugfixes. While a bugfix is not 
+really critical as long as the bug do not affect your software, open security issues can end in horrible 
+problems.
+While this option provides several strategies and mostly at least one of them can be adopted to the general 
```

Comment:
Alternativer (Ab-)Satz:
`The three options presented above outline different strategies. Which one to choose depends on many factors such as importance of project, impact of failure, budget, company policy and many more. This new release train brings a great change to the Java environment and while many companies, ops and devs will struggle at first there is a reason behind this change. One of Oracles designated goals with the new release trains is to deliver new language features faster.`

Reply:
done